server/src/main/java/org/cloudfoundry/identity/uaa/provider/saml/LoginSamlEntryPoint.java

```diff
@@ -36,6 +46,30 @@ public void setProviderDefinitionList(SamlIdentityProviderConfigurator providerD
     }
 
     @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
+
+        FilterInvocation fi = new FilterInvocation(request, response, chain);
+
+        if (!processFilter(fi.getRequest())) {
+            chain.doFilter(request, response);
+            return;
+        }
+
+        try {
+            commence(fi.getRequest(), fi.getResponse(), null);
+        } catch (ServletException e) {
+            if(e.getRootCause() instanceof MetadataProviderException) {
```

Comment:
Is this the only time a MetadataProviderException is thrown? when the binding is not supported? 

Is it possible that a MetadataProviderException can be thrown for other reasons, at which point maybe we should log the error, at least in debug mode?